### ISSUE TITLE
Direct requests to resource annotations do not load the resource correctly

### ISSUE THREAD

**Issue by tsmyre** [ManifoldScholar/manifold#1428]
The resource in question is loaded in the SSR, but disappears when the client rendering takes over:

https://edge.manifoldapp.org/read/c49468dc-8c9b-4076-8f46-03bc3dd29510/section/8a5e78ec-deaa-4673-8d6e-1a59412a75cf/resource/c9c89607-e8a8-414d-a3f7-9622c7a91b9a

**Comment by SMaxOwok** [comment]
```
warning.js?da67:33 Warning: Prop `dangerouslySetInnerHTML` did not match. Server: "" Client: "<p>A link to the Newspapers text in the Manifold reader with attendant collection of related resources:</p>\n\n<blockquote>\n<p>NEWSPAPERS. Judge Longstreet says: “Small is the sum that is required to patronize a newspaper and most amply remunerated is the patron. I care not how humble and unpretending the gazette which he takes it is next to impossible to fill a sheet fifty-two times a year without putting into it something that is worth the subscription price. Every parent whose son is off to school should be supplied with a newspaper and most amply remunerated is the patron. I well remember what a difference there was between those of my schoolmates who had and those who had not access to newspapers. Other things being equal the first were always decidedly superior to the last in debate and composition at least. The reason is plain—they had command of more facts. A newspaper is a history of current events as well as a curious and interesting miscellany and which youth will peruse with delight when they will read nothing else.”</p>\n</blockquote>"
```

**Comment by SMaxOwok** [comment]
This was fixed in 702a3bcf39abccb76c8d234537bb420ca81c4aaf
